qa/L0_lifecycle/test.sh

```diff
@@ -1705,6 +1711,117 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+LOG_IDX=$((LOG_IDX+1))
+
+# LifeCycleTest.test_concurrent_load_speedup
+rm -rf models
+mkdir models
+cp -r identity_zero_1_int32 models && mkdir -p models/identity_zero_1_int32/1
```

Comment:
This is a bit difficult to read. Could you save `identity_zero_1_int32` as a variable to make it easier to see what's happening?

Reply:
Same for other variables that could simplify things below, like `python_identity_fp32`.

Reply:
Update commit: 
[Move test model name into variable](https://github.com/triton-inference-server/server/pull/5360/commits/771d2217df343eff3d55c886bebb99cf10a3d3a2)